packages/indexer-common/src/allocations/query-fees.ts

```diff
@@ -89,6 +121,9 @@ export class AllocationReceiptCollector implements ReceiptCollector {
   declare voucherRedemptionBatchThreshold: BigNumber
   declare voucherRedemptionMaxBatchSize: number
   declare protocolNetwork: string
+  declare tapSubgraph: TAPSubgraph
+  declare finalityTime: number
+  declare sequelize: Sequelize
```

Comment:
Any ideas here? Tried bringing in just QueryInterface instead of entire Sequelize package?

Reply:
Yes this has been changed since:
https://github.com/graphprotocol/indexer/blob/gusinacio/collect-tap-ravs/packages/indexer-common/src/allocations/query-fees.ts#L38